src/igvfd/audit/treatment.py

```diff
@@ -12,12 +12,12 @@
 def audit_treatment_term_id_check(value, system):
     '''
         audit_detail: Treatments with a newly requested term ID are expected to have it updated following its addition to the appropriate ontology database.
-        audit_category: treatment term has been newly requested
+        audit_category: NTR treatment term id
```

Comment:
Do you need the id on the end?

Reply:
I think it's more specific and would be informative to users unfamiliar with "NTR".

Reply:
Though I guess this is an internal action audit so they wouldn't even see this audit.

Reply:
capitalize ID